### PR TITLE
fix could not find ESRGAN4X problem

### PR DESCRIPTION
**Pull request by wangzijian1010**
the name of ESRGAN4X method has chaned in WEBUI, need update the name of webuiapi's name of ESRGAN

### CODE REVIEW

**Comment by davidmartinrius** [comment]
Hi! How are you doing?

Why did you delete the lines of the function "util_get_current_model" from 779 to 785?

I see you finally return "return self.get_options()["sd_model_checkpoint"]"

Thanks,

David Martin Rius

**Comment by wangzijian1010** [comment]
> Hi! How are you doing?
> 
> Why did you delete the lines of the function "util_get_current_model" from 779 to 785?
> 
> I see you finally return "return self.get_options()["sd_model_checkpoint"]"
> 
> Thanks,
> 
> David Martin Rius

hi there, it's my mistake, i forget to undo this delte

**Comment by davidmartinrius** [comment]
ok @wangzijian1010 , anyway, there is an endpoint to list the available upscalers in /sdapi/v1/upscalers

Also there is an already implemented function for that get_upscalers()

So, the hardcoded upscalers need to dissapear and this pull request is not needed anymore.

Thank you!

David Martin Rius

**Comment by davidmartinrius** [comment]
Well, after reviewing it again it makes sense to have that properties hardcoded in webuiapi.py. Although they can become obsolete at any time. It's good to have them available. And if one stops working it is always possible to call the get_upscalers endpoint and update the list by hand.

If you restore the function util_get_current_model I can add my review to this pull request.

Thank you!

**Comment by wangzijian1010** [comment]
> Well, after reviewing it again it makes sense to have that properties hardcoded in webuiapi.py. Although they can become obsolete at any time. It's good to have them available. And if one stops working it is always possible to call the get_upscalers endpoint and update the list by hand.
> 
> If you restore the function util_get_current_model I can add my review to this pull request.
> 
> Thank you!

hi there!, thank you david, I have restore line 779 to 785 and you can add your review to this pull request, thanks for your reminder, i see your github repo has a lot of ai, can I add you on Skype?

File: webuiapi/webuiapi.py [hunk 1]
```diff
@@ -783,6 +783,7 @@ def util_get_current_model(self):
             sd_models = self.get_sd_models()
             sd_model = [model for model in sd_models if model["sha256"] == options["sd_checkpoint_hash"]]
             return sd_model[0]["title"]
+        return self.get_options()["sd_model_checkpoint"]
```
Comment (davidmartinrius) [inline webuiapi/webuiapi.py:786]: I think this line should not be here

**Review by davidmartinrius — CHANGES_REQUESTED** [review]
Remove extra line in function util_get_current_model(self)

File: webuiapi/webuiapi.py (hunk 1 above)
Comment (wangzijian1010) [reply webuiapi/webuiapi.py:786]: ok i forgot it,

**Comment by davidmartinrius** [comment]
> > Well, after reviewing it again it makes sense to have that properties hardcoded in webuiapi.py. Although they can become obsolete at any time. It's good to have them available. And if one stops working it is always possible to call the get_upscalers endpoint and update the list by hand.
> > If you restore the function util_get_current_model I can add my review to this pull request.
> > Thank you!
> 
> hi there!, thank you david, I have restore line 779 to 785 and you can add your review to this pull request, thanks for your reminder, i see your github repo has a lot of ai, can I add you on Skype?

Absolutely, if you wanted you can share your skype id and I will add you in my contact list. 

David Martin Rius

**Review by mix1009 — APPROVED** [review]

**Pull request merged**